.travis.yml

```diff
@@ -0,0 +1,58 @@
+language: python
+python: "3.8"
+
+pytest: &pytest
+  stage: test
+  services:
+    - postgresql
+  addons:
+    postgresql: $PG_VERSION
+  before_install:
+    - sudo apt-get update
+    - sudo apt-get --yes remove postgresql\*
+    - sudo apt-get install -y postgresql-${PG_VERSION} postgresql-client-${PG_VERSION}
+    - sudo cp /etc/postgresql/{9.6,$PG_VERSION}/main/pg_hba.conf
+    - sudo service postgresql restart $PG_VERSION
```

Comment:
Do czego to jest używane?

Reply:
TravisCI does not support higher than PG10 version too well so we need to manually install it